src/main/java/com/epam/test_generator/controllers/caze/CaseTransformer.java

```diff
@@ -66,7 +69,13 @@ public Case updateFromDto(CaseUpdateDTO dto, Case caze) {
             caze.setStatus(dto.getStatus());
         }
         if (dto.getTags() != null) {
-            caze.setTags(dto.getTags().stream().map(tagTransformer::fromDto).collect(Collectors.toSet()));
+            if (dto.getTags().size() == 0) {
+                caze.setTags(new HashSet<>());
+            } else if (caze.getTags() != null) {
+                dto.getTags().stream().map(tagTransformer::fromDto).forEach(newTag -> caze.getTags().add(newTag));
```

Comment:
Maybe we could add `addTag` ,`addTags`, `removeTags` methods for `Case` and `Suit` and use these methods instead of `getTags()` ?

Reply:
Done. Moved this logic to Taggable interface. Now Suit and Case implement Taggable.